.github/workflows/ci.yml

```diff
@@ -101,22 +101,26 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-${{ matrix.compiler }}-${{matrix.build_type}}-${{matrix.generator}}-${{matrix.developer_mode}}
 
+      - uses: actions/setup-python@v2
+
+      - run: pip3 install cmake ninja conan gcovr
```

Comment:
This is incomplete. It does not put the paths that conan and gcovr are called from on the path. Have you tried this inside Docker or locally?? 

https://github.com/aminya/setup-cpp/blob/94827f017e0e4445837e4cc099daf7ce849a059c/src/utils/setup/setupPipPack.ts#L35-L56

Reply:
> > > It is not setup-cpp's fault that the pre-installed Python broke on Windows images. In fact setup-cpp does not call "setup-python" until it actually needed. I do not approve of this, as this is like discarding all the things it was considered in setup-cpp and trying to re-implement them in bash.
> > 
> > 
> > It is not discarding everything setup-cpp is used for. Setup-cpp still does all of the hard work of finding and installing compilers across compilers.
> 
> Here I specifically meant the pip installations. As I mentioned in the above comments, it is needed to check if:
> 
>     * python paths are added to PATH on all platforms
> 
>     * if pip is installed and is up to date

It's good that setup-cpp is not fixed, but I think it's worth mentioning that all of this worked without any other path manipulations. All tests passed and even properly posted coverage reports.

Reply:
I think that what @aminya’s not saying here is that setup-cpp works just as well on a self-hosted runner. The setup-Python action takes a little more work for that. Some users depend on being able to build their projects with a self-hosted runner; maybe we should try to support them.

Reply:
We could add additional code here to make self-hosted runners work; there are directions on how to do that here: https://github.com/actions/setup-python.

Or, we could just use setup-cpp.